### PR TITLE
fix: JOIN-29724 auto login

### PR DESCRIPTION
**Pull request by MichalBaran94**
[JOIN-29724]

After user visit `login/auth` page then there will be an attempt to login user automatically.

https://user-images.githubusercontent.com/110449633/228224181-3eeb2cb2-9782-4c90-869d-325642013552.mov

[JOIN-29724]: https://joinsolutionsag.atlassian.net/browse/JOIN-29724?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by pavnaumenko — APPROVED** [review]
LGTM but wait for the FE review

**Comment by dmkav** [comment]
Why does it redirect to onboarding? and not to event-types/ page?

**Comment by MichalBaran94** [comment]
> Why does it redirect to onboarding? and not to event-types/ page?
Cuz it was a new user, created from the SAML service. I'll finish onboarding and update video to prevent any future confusion.

File: packages/app-store/apps.browser.generated.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@
     Don't modify this file manually.
 **/
 import dynamic from "next/dynamic";
+
```
[the remark this answers is not in the document]
Comment (MichalBaran94) [reply packages/app-store/apps.browser.generated.tsx:6]: ahh, this enters were added after i run `yarn install` - thought that I didn't commit them 😅 changing

**Review by dmkav — APPROVED** [review]

**Pull request merged**